annotation-utils/src/main/java/eu/europeana/annotation/config/AnnotationConfigurationImpl.java

```diff
@@ -55,6 +67,10 @@ public String getAnnotationBaseUrl() {
 	return new HashSet<>(Arrays.asList(subtitlesFormats));
     }
 
+	@Override
+	public String getAnnotationSubtitlesFormatsXml() {
```

Comment:
not needed, hardcoded value is sufficient, as the file is available in the resources. please remove this method

Reply:
Ok.